### ISSUE TITLE
imported generics are stripped

### ISSUE THREAD

**Issue by stropho** [Swatinem/rollup-plugin-dts#66]
```js
// create.d.ts
export default function create<S>(): import('./types').IBuilder<S>

```
```js
// index.d.ts
import c from './create'
export {c as create}

```

Expected Result
```js
interface IBuilder<S> {
// ...
}
declare function create<S>(): IBuilder<S>
```
Actual Result
```js
interface IBuilder<S> {
// ...
}
declare function create<S>(): IBuilder // <--- mising generic
```

I couldn't really find any workaround. Is this a bug or am I doing something wrong ?

**Comment by Swatinem** [comment]
> Is this a bug or am I doing something wrong ?

Definitely a bug. I have some things to catch up to, I will take a look at this ASAIC, I do have an idea what could be the reason for this.

**Comment by Swatinem** [comment]
@stropho should be fixed in `1.1.11`.

**Comment by stropho** [comment]
Great, thanks a lot!
